lib/config/recommended.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     'deprecated-inline-view-helper': true,
     'deprecated-render-helper': true,
     'simple-unless': true,
-    'unused-block-params': true
+    'unused-block-params': true,
+    'no-attr-in-components': true,
```

Comment:
Please do not update the default config in this PR (adding a new default rule is a breaking change, so it can't get released within the current minor version). Can you remove this entry?